Components/Avatar/avatar.css

```diff
@@ -0,0 +1,73 @@
+.avatar-sizes-list li
+{
+    font-size: 1rem;
+}
+
+.img-avatar-container
+{
+    width: 100%;
+    height: 150px;
+    box-shadow: rgba(0, 0, 0, 0.02) 0px 1px 3px 0px, rgba(27, 31, 35, 0.15) 0px 0px 0px 1px;
+    background-color: #F5F7F9;
+    display: flex;
+    justify-content: space-evenly;
+    align-items: center;
+    flex-wrap: wrap;
+    padding: 1rem;
+    box-sizing: border-box;
+}
+
+.avatar
+{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    color: var(--background-color);
+    background-color: var(--primary-dark-blue-color);
+    border-radius: 50%;
+}
+
+.avatar-x-large
+{
+    font-size: 4em;
```

Comment:
I think here you should use rems instead of em

Reply:
Done, changed to rems